tests/console/sshd.pm

```diff
@@ -70,7 +70,15 @@ sub run {
     }
 
     # Restart sshd and check it's status
-    systemctl 'restart sshd';
+    my $ret          = systemctl('restart sshd', ignore_failure => 1);
+    my $fips_enabled = script_output('cat /proc/sys/crypto/fips_enabled') eq '1';
+
+    # If restarting sshd service is not successful and fips is enabled, we have encountered bsc#1189534
+    if (($ret != 0) && ($fips_enabled)) {
```

Comment:
Soft failing a critical test like this should IMO be restricted to specific versions. So I suggest

`if (($ret != 0) && $fips_enabled && is_sle("<=15-SP2")) {`

Reply:
good point

Reply:
PR that addresses this point: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13096